json-simple/1.1.1.wso2v1/pom.xml

```
@@ -0,0 +1,66 @@
+<!--
+~ Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+~
+~ WSO2 LLC. licenses this file to you under the Apache License,
+~ Version 2.0 (the "License"); you may not use this file except
+~ in compliance with the License.
+~ You may obtain a copy of the License at
+~
+~ http://www.apache.org/licenses/LICENSE-2.0
+~
+~ Unless required by applicable law or agreed to in writing,
+~ software distributed under the License is distributed on an
+~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+~ KIND, either express or implied. See the License for the
+~ specific language governing permissions and limitations
+~ under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.googlecode.json-simple.wso2</groupId>
```

We need to change this to the new format. We don't use the wso2 suffix anymore right?
```suggestion
    <groupId>org.wso2.orbit.com.googlecode.json-simple</groupId>
```

---

fixed with https://github.com/wso2/orbit/pull/913/commits/66038efa4f7dab52a09789a39cc8cde8bb0145a7